moveit_ros/planning_interface/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -162,16 +163,16 @@ class MoveItCpp
   bool execute(const std::string& group_name, const robot_trajectory::RobotTrajectoryPtr& robot_trajectory,
                bool blocking = true);
 
-protected:
-  std::shared_ptr<tf2_ros::Buffer> tf_buffer_;
-  std::shared_ptr<tf2_ros::TransformListener> tf_listener_;
-
 private:
   //  Core properties and instances
-  ros::NodeHandle node_handle_;
+  rclcpp::Node::SharedPtr node_;
```

Comment:
It may be better to save the node by it's components rather than the full node_. That seems to be the approach used by all non-executables

Reply:
Again, what's the purpose? We are currently sharing the node with all member classes and the use is pretty diverse, so I don't see any good way to restrict the parameters. This belongs to node lifetime/handling efforts we didn't consider for this port.

Reply:
Fair enough